tests/jeos/firstrun.pm

```diff
@@ -50,6 +50,16 @@ sub verify_user_info {
     # User has locale defined in firstboot, root always defaults to POSIX (i.e. English)
     my $proglang = $args{user_is_root} ? 'en_US' : $lang;
     assert_script_run("ldd --help | grep '^" . $lang_data{$proglang} . "'");
+
+    return if $user;
+
+    if (script_run('rpm -q system-group-wheel') == 0) {
+        if (script_run('getent group wheel') && is_sle('16+')) {
```

Comment:
This is faster on SLE < 16 as no command would be run:

```suggestion
        if (is_sle('16+') && script_run('getent group wheel')) {
```

Reply:
That is true, but the `getent group wheel` is the core check. I believe it would be better to see the result of that command anyway, in case of further debug. And we are dealing with a ncurses test, speed is not a problem :D